### ISSUE TITLE
CLI create-package command

### ISSUE THREAD

**Issue by Jomy10** [chinedufn/swift-bridge#37]
Alright, so the last piece of the puzzle for Swift Packages would be the CLI. 

Since the cli crate is currently empty, I suggest we use [Clap](https://docs.rs/clap/latest/clap/). I've used this before, and it's really easy to set up and it takes care of a lot of the boilerplate for a CLI. There are a lot of [examples](https://github.com/clap-rs/clap/tree/master/examples/tutorial_builder#quick-start) to draw from as well.

In the previous issue, you commented this as a suggestion for the CLI:
```bash
# In Bash
swift-bridge generate-package \
  --bridges-dir some-out-dir \
  --ios target/aarch64-apple-ios/debug/libmy_rust_lib.a \
  --macos target/x86_64-apple-darwin/debug/libmy_rust_lib.a \
  --out-dir ./my-swift-package
```

**Comment by chinedufn** [comment]
Nice, I use Clap as well.

Maybe `create-package` instead of `generate-package` but otherwise that all sounds great to me.

**Comment by chinedufn** [comment]
Closed by #38
